test/integration/model.test.js

```diff
@@ -2843,5 +2843,29 @@ describe(Support.getTestDialectTeaser('Model'), () => {
       const users1 = await user.findAll();
       expect(users1[0].username).to.equal('jon');
     });
+
+    it('should correctly set identifiers in a column with autoIncrement with bigint values', async function () {
+      if (dialectName === 'sqlite') {
+        return;
+      }
+
+      const user = this.sequelize.define('User', {
```

Comment:
To follow our conventions (it's not respected in old code but we're slowly refactoring everything)

```suggestion
      const User = this.sequelize.define('User', {
```